packages/terra-table/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Unreleased
 4.8.0 - (June 9, 2020)
 ------------------
 ### Changed
+* Rows and Header checkbox when disabled are now accessible for screenreaders to read out accordingly.
```

Comment:
Move this log to the "Unreleased" section

Reply:
updated